crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -224,6 +230,22 @@ impl RealRibosome {
     }
 
     pub fn module(&self, zome_name: &ZomeName) -> RibosomeResult<Arc<Module>> {
+        if holochain_wasmer_host::module::SERIALIZED_MODULE_CACHE
+            .get()
+            .is_none()
+        {
+            holochain_wasmer_host::module::SERIALIZED_MODULE_CACHE
+                .set(RwLock::new(SerializedModuleCache::default_with_cranelift(
```

Comment:
i know this is API doesn't live here but just leaving my suggestion here in case you'd want to change it there and here as well

```suggestion
                .try_set(RwLock::new(SerializedModuleCache::default_with_cranelift(
```

Reply:
```
pub static SERIALIZED_MODULE_CACHE: OnceCell<RwLock<SerializedModuleCache>> = OnceCell::new();
```

once cell doesn't have a `try_set` afaik, it only has `set`